.github/workflows/ci.yml

```
@@ -29,7 +29,7 @@ jobs:
           
       - name: Build and test
         run: |
-          sbt -v +test scripted
+          sbt -v +test mimaReportBinaryIssues scripted
```

Assuming you want to check for all Scala versions.

```suggestion
          sbt -v +test +mimaReportBinaryIssues scripted
```

---

Yes, makes sense. But I do wonder in which scenario a check for one version could fail, but not for the other, when there are no version-specific sources?

---

Lots of obscure scenarios :) if all scala versions emitted identical binary APIs, then we wouldn't need to do all this cross-building in the first place! There are differences in the encodings that sometimes mean a binary-compatible change in one version is not a binary-compatible change in another version.